src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import { FC, memo } from 'react';
+import './Footer.scss';
+import classNames from 'classnames';
+import { FooterFilterLink } from '../FooterFilterLink';
+import { FilteringMethod } from '../../types/FilteringStatus';
+import { CountPerStatus } from '../../utils/functions';
+
+type Props = {
+  filterStatus: FilteringMethod;
+  onStatusChange: (status: FilteringMethod) => void;
+  countPerStatus: CountPerStatus;
+  onClearAll: () => void;
+};
+
+export const Footer: FC<Props> = memo(({
+  filterStatus,
+  onStatusChange,
+  countPerStatus,
+  onClearAll,
+}) => {
+  const {
+    active,
+    completed,
+  } = countPerStatus;
+
+  const itemOrItems = active === 1 ? 'item' : 'items';
```

Comment:
Let's make it more realistic
In real projects - such problem (with plural ending) occuring often, so usually for that purpose we creating common helper function that will return correct plural ending according to items amount